internal/slave/service.go

```diff
@@ -65,25 +66,28 @@ type slaveService struct {
 	clusterInfo discovery.ClusterInfoGetter
 	isClosed    bool
 	replInfo    *replInfo
+	healthCheckPeriod uint8
 }
 
 // NewService creates a slave DKVService that periodically polls
 // for changes from master node and replicates them onto its local
 // storage. As a result, it forbids changes to this local storage
 // through any of the other key value mutators.
 func NewService(store storage.KVStore, ca storage.ChangeApplier, lgr *zap.Logger,
-	statsCli stats.Client, regionInfo *serverpb.RegionInfo, replConf *ReplicationConfig, clusterInfo discovery.ClusterInfoGetter) (DKVService, error) {
+	statsCli stats.Client, regionInfo *serverpb.RegionInfo, replConf *ReplicationConfig,
+	clusterInfo discovery.ClusterInfoGetter, healthCheckPeriod uint8) (DKVService, error) {
```

Comment:
`statsCli stats.Client, healthCheckPeriod uint8` etc could be moved to a struct to cleanup the signature.

Reply:
Created a separate struct serveropts